server/routes/users.ts

```diff
@@ -144,44 +100,58 @@ router.post('/users', async (req: Request, res: Response, next) => {
                         attributes: ['name'],
                     },
                 ],
-            });
-            db.Token.create({
-                userId: addedUser.id,
-                token: token,
-            });
+            }));
 
-            helper.sendVerEmail(username, req, token, 'confirmation');
+        // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+        addedUser && (await db.Token.create({ userId: addedUser.id, token }));
```

Comment:
I got this to pass `eslint server/routes/users.ts` in your branch:
```suggestion
        if (addedUser) await db.Token.create({ userId: addedUser.id, token });
```

Reply:
haha thanks for finding these.